setup.py

```diff
@@ -54,6 +54,7 @@
             'qontract-reconcile = reconcile.cli:integration',
             'e2e-tests = e2e_tests.cli:test',
             'app-interface-reporter = tools.app_interface_reporter:main',
+            'qontract-cli = tools.qontract_cli:root',
```

Comment:
Have you considered making this a sub-command of `qontract-reconcile`? The naming `qontract-cli` seems quite generic. One can think it has functionalities from all qcontract-* projects. I mean, I'm totally in for having a generic 
`qontract-cli` with functionalities from all qcontract-* projects, but this that you're proposing is just exposing some queries from the reconcile...

Reply:
as discussed, we don't know where this initiative is headed, so we'll keep it simple and hidden for the time being.